client/src/services/getAllRecords.ts

```diff
@@ -0,0 +1,21 @@
+import fetch from "node-fetch";
+
+interface IDriveFile {
+  id: string;
+  title: string;
+  output: string;
+}
+
+export type DriveFileList = Array<IDriveFile>;
+
```

Comment:
Shall we export IDriveFile and DriveFileList into a Type file as they will likely be reused? We need to inline export as discussed: so
```
export interface IDriveFile {}
export type DriveFileList
```